etc/workflows/duplicate-event.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<definition xmlns="http://workflow.opencastproject.org">
+
+  <id>duplicate-event</id>
+  <title>Duplicate Event</title>
+  <tags>
+    <tag>archive</tag>
+  </tags>
+  <description/>
+  <configuration_panel>
+    <![CDATA[
+      <div id="workflow-configuration">
+        <fieldset>
+          <input
+            id="numberOfEvents"
+            name="numberOfEvents"
+            type="number"
+            class="configField"
+            onkeypress="return event.charCode > 47"
+            oninput="checkValueInBounds()"
+            min="1"
+            value="1"
+            max="10000"
+            />
+          <label for="numberOfEvents">Number of Events</label>
+        </fieldset>
+      </div>
+
+      <script type="text/javascript">
```

Comment:
I've asked Kim to ensure that users cannot enter numbers that won't work ;-)

The workflow operation handler itself mitigates a denial-of-service attack by limiting the maximum number of duplicates it will create. This implies, however, that the front-end does this limitation, too: Otherwise, users will wonder why the workflow did not create as many duplicates as they have asked for. So this is a usability feature - not a security feature.

If this can be done easier (min and max), that is of course preferable.

Reply:
min and max only apply when the user uses the up and down arrows to set the value of the number field. He can also enter the number freely and in this case, he is able to enter negative numbers as well as numbers higher than max. At least with the browser I used for testing (I think it was chrome). 

The javascript code suppresses the "-" sign as well as numbers which exceed min/max.

Of course, this has nothing to do with security, but is a usability feature.

I think this workflow definition is just an example. Admins may customize it to fit the needs of their users.

Reply:
Ok, makes sense. Thanks.

> I think this workflow definition is just an example. Admins may customize it to fit the needs of their users.

Examples should go into the documentation, not in the default set of workflows. But I think it's ok to have this as default workflow. Though maybe we should drop max to something more reasonable like 25?